05 Lean CLI/08 Backtesting/01 Deployment/02 Run Local Backtests.html

```diff
@@ -90,7 +90,7 @@ <h4>US Equity Options Algorithms</h4>
     </li>
 
     <li>
-        <a href="https://www.quantconnect.com/docs/v2/lean-cli/projects/getting-started#02-Create-Projects">Create a new local project</a>.
+        <a href="https://www.quantconnect.com/docs/v2/lean-cli/projects/project-management#02-Create-Projects">Create a new local project</a>.
```

Comment:
If a link in the CLI docs points to a section in the CLI docs, it should start with /docs/v2 ... 
Otherwise, if someone is on the lean.io site, they will be redirected to quantconnect.com